config/config.go

```diff
@@ -1,6 +1,52 @@
 package config
 
-import "golbat/geo"
+import (
+	"fmt"
+	"golbat/geo"
+)
+
+type WebhookType uint8
+
+// PayloadType returns the string to be used in the webhook payload for this type.
+func (t WebhookType) PayloadType() string {
+	return webhookTypeToPayloadType[t]
+}
+
+const (
```

Comment:
It's a shame these are here and not in the webhook module - feels like it's in the wrong place and the master definition should be in the webhook sender.

Reply:
Yes, it is a shame, I think.

I don't really like this type and constants being in 'config' either. But it looks like 'config' is supposed to be validating the config. There's evidence from existing panic()s. So, I added that in here. 'config' needs to know the possible values in order to do that.

So, we can put them in 'webhooks' where they belong, but then 'config' would need to import 'webhooks' to know the possible values to allow (unless we just want to duplicate them all). But it's a shame that 'config' cannot import 'webhooks'. :) 'webhooks' imports 'config'.

I have a few thoughts on how to redo this. I'm going to sleep soon and will revisit this later and come back with something better. Suggest ideas in the meantime, if you want.

Thnx for calling this out.